### ISSUE TITLE
Ban Realname/Gecos from connecting to the network

### ISSUE THREAD

**Issue by Casey952** [inspircd/inspircd#1426]
It would be useful to be able to ban by realname/gecos and in the conf from connecting to the network.

ban by `*name*` with wildcards.

To be able to stop bots and things that have a certain name in its realname.

**Comment by SISheogorath** [comment]
This is already possible by using the rline module.

The only missing thing is the config option.

https://github.com/inspircd/inspircd/blob/25c1319e0d16f4cea707a3b7968b70530a5a333f/docs/conf/modules.conf.example#L1515-L1538

**Comment by GTAXL** [comment]
Agreed, would love to see a config option to do this instead of relying on R-LINE and xline_db to do something Unreal offers in the config out of the gate. While it is rare I ban by real name, a common entry I put in my config is "Nmap Wuz Here" which blocks 99.9% of nmap scans from collecting information from the IRCd, as nobody bothers to modify the nmap script.
